src/platform/plugins/shared/dashboard/server/dashboard_saved_object/migrations/dashboard_saved_object_migrations.ts

```diff
@@ -35,6 +37,10 @@ export const createDashboardSavedObjectTypeMigrations = (
     migrateByValueDashboardPanels
   ) as MigrateFunctionsObject;
 
+  const controlsMigrations = deps.controls.getAllMigrations();
+
+  const dependencyMigrations = mergeMigrationFunctionMaps(embeddableMigrations, controlsMigrations);
```

Comment:
So if I'm following correctly: this refactor is split `controlsMigrations` from `embeddableMigrations` and so the end result for dashboards is the same...

Reply:
Exactly!